Q/QuantumEspresso/build_tarballs.jl

```diff
@@ -51,9 +51,19 @@ platforms = filter!(!Sys.isapple,   platforms)
 # The products that we will ensure are always built
 products = [
     ExecutableProduct("pw.x", :pwscf),
+    ExecutableProduct("bands.x", :bands),
+    ExecutableProduct("plotband.x", :plotband),
+    ExecutableProduct("plotrho.x", :plotrho),
+    ExecutableProduct("dos.x", :densityofstates),
+    ExecutableProduct("ibrav2cell.x", :ibrav2cell),
+    ExecutableProduct("kpoints.x", :kpoints),
     ExecutableProduct("cp.x", :carparinello),
     ExecutableProduct("ph.x", :phonon),
+    ExecutableProduct("q2r.x", :reciprocal2real),
+    ExecutableProduct("matdyn.x", :matdyn),
+    ExecutableProduct("dynmat.x", :dynmat),
```

Comment:
Well, it is weird for me to just name `dynmat` to `dynamical_matrix` but not `matdyn`. Because they both impose ASR & diagonalize dynamical matrices ([1](https://gitlab.com/QEF/q-e/-/blob/develop/PHonon/PH/matdyn.f90#L28-31) and [2](https://gitlab.com/QEF/q-e/blob/3ec392ecee8a83958789d31be5737adac07974ba/PHonon/PH/dynmat.f90#L12-19)). What about we name `dynmat` to `phonons_gamma_point`, and `matdyn` to `phonons_generic_point`? Idea from http://www.fisica.uniud.it/~giannozz/QE-Tutorial/handson_phon.html. If they sound bad, we can just leave them to `matdyn` & `dynmat`.

Reply:
But `phonons_gamma_point` & `phonons_generic_point` kind of conflicts with `phonon`. People may ask what does `phonon` do.

Reply:
I agree with your line of argument. How about `dynmat_gamma` and `dynmat_generic`? Or alternatively `dynamical_matrix_gamma` and `dynamical_matrix_generic`? That is at least sufficiently different to `phonon` and reasonably close to what they do.

Reply:
I prefer `dynamical_matrix_gamma` & `dynamical_matrix_generic`. `dynmat_gamma` & `dynmat_generic` sound like they are subcommands of `dynmat.x`.